nebula-components/components/_card.scss

```diff
@@ -0,0 +1,43 @@
+@if $nb-use-card {
+  .#{$nb-namespace}c-card {
+    background-color: $nb-card-background;
+    box-shadow: 3px 1px 5px rgba($nb-card-box-shadow-color, $nb-card-box-shadow-alpha);
```

Comment:
I'd add the padding to the card without the block.  and make the card banner have negative margin to negate the padding.